bundles/org.openhab.binding.pentair/src/main/resources/OH-INF/thing/controller.xml

```diff
@@ -0,0 +1,375 @@
+				<option value="SOLARPREFERRED">Solar Preferred</option>
+				<option value="SOLAR">Solar</option>
+			</options>
+		</state>
+
+	</channel-type>
+
+	<channel-type id="heatactive">
+		<item-type>Number</item-type>
+		<label>Heat active</label>
+		<description>Heat active state</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="setpointtype">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature set point</label>
+		<description>Temperature set point</description>
+		<state pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="lightmodetype">
+		<item-type>String</item-type>
+		<label>Light Mode</label>
+		<description>Light mode</description>
+		<state>
+			<options>
+				<option value="OFF">Off</option>
+				<option value="ON">On</option>
+				<option value="COLORSYNC">Color Sync</option>
+				<option value="COLORSWIM">Color Swim</option>
+				<option value="COLORSET">Color Set</option>
+				<option value="PARTY">Party</option>
+				<option value="ROMANCE">Romance</option>
+				<option value="CARIBBEAN">Caribbean</option>
+				<option value="AMERICAN">American</option>
+				<option value="SUNSET">Sunset</option>
+				<option value="ROYAL">Royal</option>
+				<option value="BLUE">Blue</option>
+				<option value="GREEN">Green</option>
+				<option value="RED">Red</option>
+				<option value="WHITE">White</option>
+				<option value="MAGENTA">Magenta</option>
+			</options>
+		</state>
+		<autoUpdatePolicy>recommend</autoUpdatePolicy>
+	</channel-type>
+
+	<channel-type id="scheduletype">
+		<item-type>String</item-type>
+		<label>Schedule Type</label>
+		<description>>Type of schedule (None, Normal, EggTimer, OnceOnly)</description>
+		<state>
+			<options>
+				<option value="NONE">None</option>
+				<option value="NORMAL">Normal</option>
+				<option value="EGGTIMER">Egg Timer</option>
+				<option value="ONCEONLY">Once Only</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="schedulestringtype">
+		<item-type>String</item-type>
+		<label>Schedule</label>
+		<description>String format of schedule</description>
+	</channel-type>
+
+	<channel-type id="starttype">
+		<item-type>Number</item-type>
+		<label>Start Time</label>
+		<description>Start Time in minutes</description>
+		<state min="0" max="1440"/>
+	</channel-type>
+
+	<channel-type id="endtype">
+		<item-type>Number</item-type>
+		<label>End Time</label>
+		<description>End time (or duration for Egg Timer) of schedule in minutes</description>
+		<state min="0" max="1440"/>
+	</channel-type>
+
+	<channel-type id="circuittype">
+		<item-type>Number</item-type>
+		<label>Circuit</label>
+		<description>>Circuit</description>
+	</channel-type>
+
+	<channel-type id="daystype">
+		<item-type>String</item-type>
+		<label>Days</label>
+		<description>>Days</description>
+	</channel-type>
+
+	<channel-type id="uomtype">
+		<item-type>String</item-type>
+		<label>Unit of measure (Celcius, Fahrenheit)</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
See above